src/main/java/core/basesyntax/services/FigureSupplier.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax.services;
+
+import core.basesyntax.Color;
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import core.basesyntax.model.Figure;
+import java.util.Random;
+
+public class FigureSupplier {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final int figureCeilingParam = 10;
+    private final Random random = new Random();
+
+    public Figure getDefaultFigure() {
+        Circle defaultFigure = new Circle();
+        defaultFigure.setRadius(10);
+        defaultFigure.setColor(Color.WHITE.name().toLowerCase());
+        return defaultFigure;
+    }
+
+    public Figure getRandomFigure(String figure) {
+        switch (figure) {
```

Comment:
Get a random number based on the number of pieces and use this number in the case.